### ISSUE TITLE
Strips existing code

### ISSUE THREAD

**Issue by RyanNutt** [shufo/vscode-blade-formatter#2]
Tried this on a couple of blade templates and it stripped everything up to the first blade marker `{{`. One had some HTML first and the other was a mix of HTML and PHP.

**Comment by shufo** [comment]
@RyanNutt 
Could you provide example template that occurs the problem?
I would like to try reproducing.

**Comment by RyanNutt** [comment]
Sure...

I didn't catch it before, but both files that I tried this on had a PHP code block on the top. One, like this, with comments. The other had a bit of code that sets up variables. If I take out the PHP it appears to format correctly. 

Before
```php
<?php
/* Some comments on this template
 */
?>
<div class="font-ext-links">
    <a class="btn btn-cta" href="{{ url('download/' . $font->slug) }}" title="Download {{ $font->title }}">
        <i class="fa fa-fw fa-download"></i>
        Download
    </a>

</div>
```

After
```php
{{ url('download/' . $font->slug) }}" title="Download {{ $font->title }}">
<i class="fa fa-fw fa-download"></i>
Download
</a>

</div>
```

**Comment by shufo** [comment]
@RyanNutt 
Thanks for sharing!
Also reproduced in my environment.

I'll try figure out what causes this behaviour.

Current workaround is to comment by blade syntax or remove php code block section will generate constant results.

```php
{{-- Some comments on this template --}}
<div class="font-ext-links">
    <a class="btn btn-cta" href="{{ url('download/' . $font->slug) }}" title="Download {{ $font->title }}">
        <i class="fa fa-fw fa-download"></i>
        Download
    </a>

</div>
```

or 

```php
<div class="font-ext-links">
    <a class="btn btn-cta" href="{{ url('download/' . $font->slug) }}" title="Download {{ $font->title }}">
        <i class="fa fa-fw fa-download"></i>
        Download
    </a>

</div>
```

**Comment by shufo** [comment]
Just published [0.3.1](https://marketplace.visualstudio.com/items/shufo.vscode-blade-formatter/changelog) with this bug fix.
